src/app/branded/step-1/branded-checkout-step-1.component.js

```diff
@@ -40,7 +40,7 @@ class BrandedCheckoutStep1Controller {
       (this.itemConfig.CAMPAIGN_CODE.match(/^[a-z0-9]+$/i) === null || this.itemConfig.CAMPAIGN_CODE.length > 30)) {
       this.itemConfig.CAMPAIGN_CODE = ''
     }
-    this.itemConfig['campaign-page'] = this.campaignPage
+    this.itemConfig.CAMPAIGN_PAGE = this.campaignPage
```

Comment:
This shouldn't change. It is not part of what gets sent to ElasticPath, only used on the front-end.